### PR TITLE
Agentapp 65

### PR DESCRIPTION
**Pull request by vishnuscv**

### CODE REVIEW

File: src/main/java/com/mifos/api/datamanager/DataManagerAuth.java [hunk 1]
```diff
@@ -36,4 +36,10 @@ public DataManagerAuth(BaseApiManager baseApiManager) {
         return baseApiManager.getAuthApi().setFcmToken(userId, new FcmToken(token));
     }
 
+    public Observable<String> updatePassword(int userId, String password, String repeatPassword) {
+        UpdatePasswordPayload updatePasswordPayload = new UpdatePasswordPayload();
```
Comment (Ezcred) [inline src/main/java/com/mifos/api/datamanager/DataManagerAuth.java]: make this class elements final and pass in constructor.

File: src/main/java/com/mifos/api/model/UpdatePasswordPayload.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.mifos.api.model;
+
```
Comment (Ezcred) [inline src/main/java/com/mifos/api/model/UpdatePasswordPayload.java:2]: remove one line

File: src/main/java/com/mifos/api/model/UpdatePasswordPayload.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+import lombok.Data;
+
+@Data
+public class UpdatePasswordPayload {
+  private String password;
```
Comment (Ezcred) [inline src/main/java/com/mifos/api/model/UpdatePasswordPayload.java]: make it final

File: src/main/java/com/mifos/api/services/AuthService.java [hunk 4]
```diff
@@ -21,9 +22,15 @@
 public interface AuthService {
 
     @POST(APIEndPoint.AUTHENTICATION)
-    Observable<User> authenticate(@Query("username") String username,
-                                           @Query("password") String password);
+    Observable<User> authenticate(
+      @Query("username") String username,
```
Comment (Ezcred) [inline src/main/java/com/mifos/api/services/AuthService.java]: add 2 space more.

File: src/main/java/com/mifos/api/model/UpdatePasswordPayload.java (hunk 2 above)
Comment (vishnuscv) [reply src/main/java/com/mifos/api/model/UpdatePasswordPayload.java:2]: done

File: src/main/java/com/mifos/api/model/UpdatePasswordPayload.java (hunk 3 above)
Comment (vishnuscv) [reply src/main/java/com/mifos/api/model/UpdatePasswordPayload.java]: done

File: src/main/java/com/mifos/api/datamanager/DataManagerAuth.java (hunk 1 above)
Comment (vishnuscv) [reply src/main/java/com/mifos/api/datamanager/DataManagerAuth.java]: done

File: src/main/java/com/mifos/api/services/AuthService.java (hunk 4 above)
Comment (vishnuscv) [reply src/main/java/com/mifos/api/services/AuthService.java]: done

**Review by vishnuscv — COMMENTED** [review]
done fixes

**Pull request merged**